arch/arm64/core/xen/hypercall.S

```diff
@@ -25,3 +25,5 @@ HYPERCALL(event_channel_op);
 HYPERCALL(hvm_op);
 HYPERCALL(memory_op);
 HYPERCALL(domctl);
+HYPERCALL(xen_version);
+HYPERCALL(sysctl);
```

Comment:
Please, cover this with `CONFIG_XEN_INITIAL_DOMAIN`, we missed the same for domctl, it will be added in `zephyr-v3.3.0-xt` branch.

Reply:
should I rebase this to v3.3.0-xt?